docs/docs/about/modelling/setup/models/compressor_modelling/compressor_models_types/single_speed_compressor_train_model.md

```diff
@@ -13,8 +13,9 @@ This means that a single speed compressor model needs the following to be define
 
 - A polytropic compressor chart for every compressor stage in the compressor train. For single speed trains, eCalc
   only supports user defined single speed compressor charts.
-- A [FLUID_MODEL](/about/references/FLUID_MODEL.md).
-- A [PRESSURE_CONTROL](/about/references/PRESSURE_CONTROL.md).
+- [CONTROL_MARGIN](/about/references/CONTROL_MARGIN.md) and [CONTROL_MARGIN_UNIT](/about/references/CONTROL_MARGIN_UNIT.md) for each compressor stage in the train
```

Comment:
I would prefer to indicate that this was added in `Since v8.26` or `Added in v8.26`. As inspired by Python docs; https://docs.python.org/3/library/warnings.html#warnings.warn_explicit (Changed in version 3.6: Add the source parameter.) Could be after/before the section, but clarificaiton that it has not always been there ...

Reply:
Updated with version description.